react/classes-optional/mentors.md

```diff
@@ -0,0 +1,36 @@
+# Mentor's Notes - React Class Components
+
+- Class components section
+  - Students often get tripped on converting between functional & class components
```

Comment:
I did a blog post a while ago about converting Classes to function components. I'll share it here incase there is any reason to incorporate it as a link.

https://dev.to/subdeveloper/migrating-from-class-components-to-react-hooks-4lp3

Reply:
Thanks I added the link to the further reading at the bottom of the lesson.